### PR TITLE
Custom Questions: Implemented with test

### PR DESCRIPTION
**Pull request by aramis-eb**
Implemented with test

### CODE REVIEW

File: search_events_app/services/filters/features/repeating_events_filter.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 class RepeatingEventsFilter(Filter):
-	CODE = "RE"
+	CODE = 'RE'
```
Comment (julianrofrano-eb) [inline search_events_app/services/filters/features/repeating_events_filter.py:5]: you should add this in another as constants, and import wherever you need them. and this attribute should be in lower case

File: search_events_app/tests/services/test_custom_questions_filter.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from search_events_app.services.filters.features import CustomQuestionFilter
+
+
+class TestCustomQuestionFilter(TestCase):
+
+    def setUp(self):
+        self.custom_question_filter = CustomQuestionFilter()
+
+    def test_apply_custom_question_filter(self):
+        features_codes = ['CQ', 'EB']
```
Comment (julianrofrano-eb) [inline search_events_app/tests/services/test_custom_questions_filter.py:14]: you can use those constants here

File: search_events_app/services/filters/features/repeating_events_filter.py (hunk 1 above)
Comment (aramis-eb) [reply search_events_app/services/filters/features/repeating_events_filter.py]: Done!

File: search_events_app/utils/feature_codes.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class FeatureCodes:
```
Comment (julianrofrano-eb) [inline search_events_app/utils/feature_codes.py:1]: you dont need to create a class for this, just a const with upper case

**Pull request merged**